Snowdog_SolrNavigation/templates/navigation/visual_swatch.phtml

```diff
@@ -2,76 +2,95 @@
 /** @var \Snowdog\SolrNavigation\Block\Navigation\ListItem $block */
 ?>
 
-<li class="
+<li
+    class="
         dropdown-list__item
         dropdown-list__item--collapse
         filter
     "
 >
     <?php if (strtolower($block->getAttributeLabel()) == 'color') : ?>
-        <div class="dropdown-list__label filter__label" aria-controls="color">
-            <?= $block->getAttributeLabel() ?>
-            <svg class="icon dropdown-list__icon">
+        <button
+            type="button"
+            class="dropdown-list__label filter__label"
+            aria-controls="filter-color"
+            aria-expanded="false"
+        >
+            <h3 class="filter__label-title">
+                <?= $block->getAttributeLabel() ?>
+            </h3>
+            <svg class="icon dropdown-list__icon" role="img">
+                <title><?= __('Arrow down') ?></title>
                 <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#angle-down') ?>"></use>
             </svg>
-        </div>
+        </button>
         <div
-            id="color"
+            id="filter-color"
             class="dropdown-list__content filter__content"
-            data-content="color"
             aria-hidden="true"
         >
-            <div class="filter__color">
+            <ul class="list filter__list">
                 <?php foreach ($block->getItems() as $_item): ?>
-                    <a
-                        href="<?= $block->getNavigationUrl($block->getAttributeCode(), $_item['value_id']) ?>"
-                        class="
-                            filter__color-option
-                            <?= $block->isActive($_item['value_id']) ? 'filter__color-option--active' : '' ?>
-                        "
-                        aria-label="<?= $_item['label'] ?>"
-                        title="<?= $_item['label'] ?>"
-                    >
-                        <div
-                            class="filter__color-sample"
-                            style="background: <?= $_item['swatch_value'] ?>"
-                        ></div>
-                        <span class="filter__color-label">
-                            <?= $_item['label'] ?>
-                        </span>
-                    </a>
+                    <li class="filter__color-item">
+                        <a
+                            href="<?= $block->getNavigationUrl($block->getAttributeCode(), $_item['value_id']) ?>"
+                            class="
+                                filter__color-option
+                                <?= $block->isActive($_item['value_id']) ? 'filter__color-option--active' : '' ?>
+                            "
+                            aria-label="<?= $_item['label'] ?>"
```

Comment:
as previous, with aria-label, the label inside span won't be read

Reply:
removed

Reply:
👍